src/payments.ts

```diff
@@ -603,6 +603,160 @@ export class Payments {
     })
   }
 
+  /**
+   *
+   * It creates a new AI Agent and a Payment Plan on Nevermined.
+   *
+   * @remarks
+   *
+   * This method is oriented to AI Builders
+   *
+   * @see https://docs.nevermined.app/docs/tutorials/builders/register-agent
+   *
+   * @param planName - The name of the Payment Plan.
```

Comment:
I think we should group the plan and agent params in 2 objects that are passed as parameter. In this implementation the interface requires more than 20 params